pkg/sidecar/sidecar.go

```diff
@@ -300,18 +235,40 @@ func (s *Sidecar) writeKey(file string, data []byte) error {
 	return ioutil.WriteFile(file, pem.EncodeToMemory(b), keyFileMode)
 }
 
-// parses a time.Duration from the the Config,
-// if there's an error during parsing, maybe because
-// it's not well defined or not defined at all in the
-// config, returns the defaultTimeout constant
-func getTimeout(config *Config) (time.Duration, error) {
-	if config.Timeout == "" {
-		return defaultTimeout, nil
+// x509Watcher is a sample implementation of the workload.X509SVIDWatcher interface
+type x509Watcher struct {
+	sidecar *Sidecar
+}
+
+// OnX509ContextUpdate is run every time an SVID is updated
+func (w x509Watcher) OnX509ContextUpdate(svids *workloadapi.X509Context) {
+	for _, svid := range svids.SVIDs {
+		w.sidecar.config.Log.Infof("SVID updated for spiffeID: %q", svid.ID)
```

Comment:
This log line is misleading. The x509 context could have been updated for myriad reasons (e.g., bundle changes, single SVID change).